packages/mwp-tracking-plugin/src/activity.js

```diff
@@ -149,12 +149,15 @@ const getOnPreResponse = cookieConfig => (request, h) => {
 		strictHeader: false, // skip strict cookie format validation (no quotes)
 	};
 
-	if (browserId) {
-		request.response.state(browserIdCookieName, browserId, FOREVER);
-	}
-	if (trackId) {
-		request.response.state(trackIdCookieName, trackId, FOREVER);
+	if (!request.response.isBoom) {
+		if (browserId) {
+			h.state(browserIdCookieName, browserId, FOREVER);
```

Comment:
`h.state` and `request.response.state` are near identical (request.response.state just returns `this`). Let's use the toolkit everywhere for consistency.

Reply:
agreed. The redundancy between the two method implementations is surprising :/

https://github.com/hapijs/hapi/blob/master/lib/response.js#L428-L432
https://github.com/hapijs/hapi/blob/master/lib/toolkit.js#L165-L168